index.bs

```diff
@@ -1111,6 +1119,8 @@ The key words "MUST", "MUST NOT", "REQUIRED", "SHALL", "SHALL NOT", "SHOULD", "S
 
         :   <dfn>rpId</dfn>
         ::  The [=Relying Party Identifier=], for the [=[RP]=] this [=public key credential source=] is [=scoped=] to.
+            This is determined by the <code>{{PublicKeyCredentialCreationOptions/rp}}.{{PublicKeyCredentialRpEntity/id}}</code>
+            parameter of the [=registration ceremony=].
```

Comment:
Perhaps this is another place we ought to use "create() operation" rather than "reg ceremony" ?

Also perhaps the reg ceremony definition ought to mention that it encompasses a "create() operation"  (similarly for authn ceremony & get() operation)?

Reply:
Thanks! I added these changes to PR #1674 (commits [3d852f2c and 42db62ad](https://github.com/w3c/webauthn/pull/1674/files/d9d608e462be5a7e19274b4b9ca9c90632054ec4..42db62ad9c42ccfe96d66c063a1f14f9c2d08606)), does that look good?

Reply:
LGTM thx :)